examples/specs/geo_point.vl.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "width": 500,
+  "height": 300,
+  "projection": {
+    "type": "albersUsa"
+  },
+  "data": {
+    "url": "data/zipcodes.csv",
+    "format": {
+      "type": "csv"
+    }
+  },
+  "transform": [{"calculate": "substring(datum.zip_code, 0, 1)", "as": "digit"}],
+  "mark": "circle",
+  "encoding": {
+    "x": {
+      "field": "longitude",
```

Comment:
I don't think they really do... field is the data field, type is how we parse it in the projection. Firld could just as easily be "long" or "lng" or "city__longitude" or "barfoo"